src/test/java/net/smartcosmos/dto/metadata/MetadataCreateTest.java

```diff
@@ -0,0 +1,54 @@
+package net.smartcosmos.dto.metadata;
+
```

Comment:
There should be a test for adding `null` and not null data.  It is a branch in the code of the MetadataCreate object.